api/interfaces/postOrgansiationPayload.ts

```diff
@@ -17,5 +17,17 @@ export interface PostOrgansiationPayload {
     dxAddress?: {
```

Comment:
Lets not name this interface which is a general thing around what is  being done with it specifically. I.e., not postOrganisationPayload but organisationPayload

Reply:
Makes sense, I've just pushed the change :)